### PR TITLE
Feature/oca 8.0 add purchase costs

### PR DESCRIPTION
**Pull request by hendrixcosta**
Feature/oca 8.0 add purchase costs

### CODE REVIEW

File: l10n_br_purchase/models/purchase.py [hunk 1]
```diff
@@ -212,7 +286,19 @@ class PurchaseOrderLine(models.Model):
         domain="[('type', '=', 'input'), ('journal_type', '=', 'purchase')]")
     fiscal_position = fields.Many2one(
         'account.fiscal.position', u'Posição Fiscal',
-        domain="[('fiscal_category_id', '=', fiscal_category_id)]")
+        domain="[('fiscal_category_id',120 '=', fiscal_category_id)]")
```
Comment (renatonlima) [inline l10n_br_purchase/models/purchase.py]: Parece que teve um erro de digitação...

**Comment by renatonlima** [comment]
@Hendrixcosta please sign OCA CLA

**Comment by rvalyi** [comment]
@Hendrixcosta we still didn't receive your CLA at the OCA.
But we need it to merge your PR here please...
cc @mileo

**Comment by rvalyi** [comment]
@Hendrixcosta any news from the OCA about your CLA? Look it's holidays in Europe and many of the OCA board members are not available and when they will they will be busy. I also heard from our team when you mate the OCA at the last Odoo Experience, so we are all to trust you and proceed with the merge, specially as you already had a commit in the project if I am correct. If you send us a mail with the CLA we can proceed with the merge while it's being sorted out at the OCA. You can send it at rvalyi_AT_akretion_DOT_com

**Comment by rvalyi** [comment]
@Hendrixcosta my bad, I asked for your CLA by email but it was alright already! In fact I mixed your case with the one of Leonardo here https://github.com/OCA/l10n-brazil/pull/361 whose CLA signature was still pending. Sorry about the confusion.
We should just review your PR then...

**Comment by rvalyi** [comment]
(correcting myself: the CLA of Leonardo is ok too now BTW...)

**Comment by mileo** [comment]
@Hendrixcosta Can you fix this conflicting files
l10n_br_purchase/models/purchase.py
l10n_br_purchase/views/purchase_view.xml

**Comment by lfdivino** [comment]
:+1:

File: l10n_br_purchase/models/purchase.py [hunk 2]
```diff
@@ -72,6 +87,62 @@ def _default_fiscal_category(self):
...
+            freight += line.freight_value
+            costs += line.other_costs_value
+            insurance += line.insurance_value
+
+        self.amount_freight = freight
+        self.amount_costs = costs
+        self.amount_insurance = insurance
+
+    amount_freight = fields.Float(
+        compute=_get_costs_value, inverse=_set_amount_freight,
```
Comment (susport) [inline l10n_br_purchase/models/purchase.py:133]: In compute method in field is not necessary attr readonly.

**Pull request closed without merging**